### PR TITLE
Add more data to `get_itn_data.R`

### LINKED ISSUE #9: Additional Datasets
We need additional datasets in the `get_itn_data.R` script run in our GHA to put on the dashboard.

### PR DESCRIPTION
**Pull request by howardbaik**
Added more datasets necessary for automating data for the dashboard in `get_itn_data.R` called in GHA:
1. ITCR Slido data (`itcr_slido_data.RDS`)
2. CRAN download stats (`cran_download_stats.tsv`)
3. OPEN meeting attendance (`open_meeting_attendance`)

### CODE REVIEW

File: .github/workflows/get_itn_data.R [hunk 1]
```diff
@@ -118,4 +118,26 @@ gh_metrics <- get_multiple_repos_metrics(repo_names = repos)
 
 saveRDS(gh_metrics, file.path(root_dir,"data", "itcr_gh_metrics.RDS"))
 
+
+# CRAN Download Stats
+download_stats <- cranlogs::cran_downloads(packages = c("ottrpal", "conrad", "ari", "text2speech"), 
+                                           # first version of ari published on 2017-08-31 
+                                           from = "2017-08-31",
+                                           to = "last-day")
```
Comment (howardbaik) [inline .github/workflows/get_itn_data.R:92]: Get package downloads from RStudio CRAN mirror

**Comment by howardbaik** [comment]
@cansavvy I am done with this PR. It is ready for your review!

**Review by cansavvy — APPROVED** [review]
Merge away!

**Comment by howardbaik** [comment]
@cansavvy I tested the GHA on this branch, but ran into an error: https://github.com/FredHutch/itn-dashboard/actions/runs/8194876721/job/22411893719#step:4:41

Does the version of metricminer on your Docker image have `get_all_ga_metrics()`?

**Comment by cansavvy** [comment]
This action changed names and is now https://hutchdatascience.org/metricminer/reference/get_multiple_ga_metrics.html

**Pull request merged**